mixer: explicit casts from void * to other pointer type

felixhaedicke opened the pull request:
g++ does not allow implicit casting of void \* to another pointer type
(at least not without -fpermissive)

felixhaedicke committed:
mixer: explicit casts from void * to other pointer type

g++ does not allow implicit casting of void * to another pointer type
(at least not without -fpermissive)

tiwai commented:
On Fri, 15 Jul 2016 01:50:14 +0200,
felixhaedicke wrote:

> g++ does not allow implicit casting of void \* to another pointer type
> (at least not without -fpermissive)
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/tiwai/salsa-lib/pull/1

Please submit the patches to alsa-devel ML.  I don't use github pull
and such features.

thanks,

Takashi

felixhaedicke committed:
Include <poll.h> instead of <sys/poll.h>

According to POSIX.1-2001, the correct include path is <poll.h>. And
when compiling against musl, it generates "deprecated" warnings, if
<sys/poll.h> is included.
